### ISSUE TITLE
Accessibility errors in Grommet components 

### ISSUE THREAD

**Issue by vijaykumar-peddamuthu** [grommet/grommet#3384]
Hi,

I was trying to updating some accessibility errors in one of our project but I am not able to correct it few components. Please find below the details of that.

1) Layer closing button (X) does not have an screen reader text. find screenshot for the same. 
![image](https://user-images.githubusercontent.com/46993098/65938966-1ef72a80-e442-11e9-8960-7e2f5be74c77.png)

2) When i use the Table header component if any header is missing the value, it's throwing the error and it's not contain any screen reader text. find screenshot for the same. 
![image](https://user-images.githubusercontent.com/46993098/65939158-b3618d00-e442-11e9-9014-9f906d0cc12a.png)

Please look at them and provide possible ways to updated the screen reader text. Above examples are exist in Grommet site also so attaching the same.

**Comment by vijaykumar-peddamuthu** [comment]
Please find below markup which I am using in our application.

![image](https://user-images.githubusercontent.com/46993098/66454313-255e5580-ea85-11e9-9e1c-6e8427e509fe.png)

**Comment by jcfilben** [comment]
For the Layer issue Wave is flagging this because you have a button with only and icon and no text. Since you only have an icon a screenreader doesn't have anything to read out for this button. You can add an `a11yTitle` to the button. 

For the table issue instead of doing `<TableHeader labels={'Name', 'Description', 'Users'} />`
Define the table headers like so:
``` javascript
<TableHeader>
  <TableRow>
    <TableCell scope="col" border="bottom">
      Name
    </TableCell>
    <TableCell scope="col" border="bottom">
      Description
    </TableCell>
    <TableCell scope="col" border="bottom">
      Users
    </TableCell>
  </TableRow>
</TableHeader>
```
